linux_os/guide/system/auditing/policy_rules/audit_delete_success/tests/no_rule.fail.sh

```diff
@@ -0,0 +1 @@
+rm -rf /etc/audit/rules.d/*.rules
```

Comment:
Here I would rather delete only the specific file from the `/etc/audit/rules.d/` - `30-ospp-v42-4-delete-success.rules`.